documentation/source/users/rmg/input.rst

```diff
@@ -469,6 +469,21 @@ For example ::
 			  branchingRatioMax=1.0,
 		)
 
+Advanced Setting: Deadend Radical Elimination (RMS Reactors only)
+-----------------------------------------------------------------
+When generating mechanisms involving significant molecular growth (such as in polymerization),
+there are many possible radicals and fast propagation pathways competing with very slow termination and chain transfer reactions.
```

Comment:
This should be "compete" not "competing"

Reply:
Disagree. 
There are reactions competing... 
Reactions compete...

Reply:
I see the issue now. More complex fix. What is the "and" joining? _Radicals_ and _reactions, or _are_ and _compete_. Once you have decided that, you can restructure the sentence to make it clearer.

Reply:
Change the "and" to ", so" and "competing" to "compete"

Reply:
@rwest Thanks for the in-person discussion today! I really like your version and I change the documentation accordingly.